$display() causes a segmentation fault when invoked on windows

Using openVAF 23_2_0 and Windows MSVC (with Desktop development C++ package) I obtain the following compilation error when using the $display function:

> error LNK2019: unresolved external symbol snprintf referenced in function cb.2

--- a/openvaf/osdi/src/lib.rs
+++ b/openvaf/osdi/src/lib.rs
@@ -209,6 +209,9 @@ pub fn compile(
         let val = cx.const_null_ptr(cx.ptr_ty(fun_ty));
         unsafe {
             llvm::LLVMSetInitializer(osdi_log, val);
+            llvm::LLVMSetLinkage(osdi_log, llvm::Linkage::ExternalLinkage);
+            llvm::LLVMSetUnnamedAddress(osdi_log, llvm::UnnamedAddr::No);
+            llvm::LLVMSetDLLStorageClass(osdi_log, llvm::DLLStorageClass::Export);
         }
 
         debug_assert!(llmod.verify_and_print());

